app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,39 @@
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/drawer_layout"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <LinearLayout
+      android:id="@+id/container_toolbar"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical">
+
+      <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar" />
+    </LinearLayout>
+
+    <FrameLayout
+      android:id="@+id/container_body"
+      android:layout_width="fill_parent"
+      android:layout_height="0dp"
+      android:layout_weight="1" />
+  </LinearLayout>
+  <fragment
```

Comment:
You have kept 1 line blank in between views, so make sure you do that everywhere. You are missing that here.